meta/classes/base.bbclass

```diff
@@ -87,6 +87,33 @@ base_update_conf_value() {
     sed -i "s%\(^${key} *[?+:]*= *\).*%\1\"${value}\"%" "${config_file}"
 }
 
+base_set_conf_value() {
+    local config_file=$1
+    local key=$2
+    local value=$3
+    local key_prefix=$4
+
+    echo "${key}${key_prefix} = "\"${value}\" >> "${config_file}"
+}
+
+base_add_conf_value() {
```

Comment:
Yocto uses append word, not add

Reply:
that's why Yocto got conflicts with function name while building dependencies